app/views/users/show.html.erb

```diff
@@ -1,10 +1,5 @@
-<p style="color: green"><%= notice %></p>
-
-<%= render @user %>
-
-<div>
-  <%= link_to "Edit this user", edit_user_path(@user) %> |
+<%= render partial: "partials/user_listing", locals: {user: @user}%>
+<%= render partial: "partials/user_details"%>
+<div class="flex justify-center m-4">
   <%= link_to "Back to users", users_path %>
-
-  <%= button_to "Destroy this user", @user, method: :delete %>
 </div>
```

Comment:
**OPTIONAL** - Please note that according to the project requirements, you are not required to have any logic in your view files, this would be done in a later milestone. I suggest having only a placeholder in this file 👍 👍